app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockScreen.kt

```diff
@@ -118,7 +118,7 @@ fun VaultUnlockScreen(
 
     // Dynamic dialogs
     when (val dialog = state.dialog) {
-        is VaultUnlockState.VaultUnlockDialog.Error -> BitwardenBasicDialog(
+        is VaultUnlockState.VaultUnlockDialog.UnlockError -> BitwardenBasicDialog(
```

Comment:
Isn't this already generic. I assume the fido2 errors need to specify a title, which we could just move into this dialog state.